docker.Makefile

```diff
@@ -110,9 +110,19 @@ lint: ## run linter(s)
 
 vendor: build_dev_image
 	$(info Update Vendoring...)
-	docker run --rm -v "$(CURDIR)":/go/src/github.com/docker/app/ $(DEV_IMAGE_NAME) make vendor
+	rm -rf ./vendor
```

Comment:
Should this be put after the container completes vendoring? (i.e. in case the container fails, your local `vendor` is gone)

Reply:
👍